gnmvidispine/vidispine_api.py

```diff
@@ -263,12 +263,20 @@ def sendAuthorized(self,method,url,body,headers):
         while True:
             self.logger.debug("sending {0} request to {1} with headers {2}".format(method,url,headers))
             try:
-                conn.request(
-                    method,
-                    url.decode('utf-8',"backslashreplace").encode('utf-8',"backslashreplace"),
-                    body.decode('utf-8',"backslashreplace") if body else None,
-                    headers
-                )
+                if rawData == False:
+                    conn.request(
+                        method,
+                        url.decode('utf-8',"backslashreplace").encode('utf-8',"backslashreplace"),
+                        body.decode('utf-8',"backslashreplace") if body else None,
+                        headers
+                    )
+                else:
+                    conn.request(
+                        method,
+                        url.decode('utf-8',"backslashreplace").encode('utf-8',"backslashreplace"),
+                        body if body else None,
+                        headers
+                    )
```

Comment:
I would suggest setting a new variable to the right body value depending on rawData and then passing that to conn.request rather than duplicating the whole call.

Reply:
Done